data/lang/module-scoop/en.json

```diff
@@ -165,7 +165,7 @@
   },
   "SUCCESS_MSG_RESCUE": {
     "description": "",
-    "message": "Thank you!"
+    "message": "Thank you very much! The crew would also like to express its thanks."
```

Comment:
In English, that would read "The crew would also like to express their thanks", as 'crew' is a plural noun referring to several persons and 'its' is a singular pronoun typically used to refer to an object, not a person